themes/purity/purity.theme.bash

```diff
@@ -14,21 +14,20 @@ STATUS_THEME_PROMPT_BAD="${bold_red}❯${reset_color}${normal} "
 STATUS_THEME_PROMPT_OK="${bold_green}❯${reset_color}${normal} "
 PURITY_THEME_PROMPT_COLOR="${PURITY_THEME_PROMPT_COLOR:=$blue}"
 
-detect_venv() {
+venv_prompt() {
 	python_venv=""
 	# Detect python venv
 	if [[ -n "${CONDA_DEFAULT_ENV}" ]]; then
 		python_venv="($PYTHON_VENV_CHAR${CONDA_DEFAULT_ENV}) "
 	elif [[ -n "${VIRTUAL_ENV}" ]]; then
 		python_venv="($PYTHON_VENV_CHAR$(basename "${VIRTUAL_ENV}")) "
+	[[ -n "${python_venv}" ]] && echo "${python_venv}"
```

Comment:
I think this line needs be *below* the `fi` in order to apply both paths.

Reply:
Yes, I noticed it after the commit, but [changed](https://github.com/Bash-it/bash-it/pull/1849/commits/64c4c30706b6b5e9ad9fb88339c7a5269f01ccb4) it right after